torch/jit/_trace.py

```diff
@@ -435,7 +435,7 @@ def run_mod_and_filter_tensor_outputs(mod, inputs, running_what):
                     + running_what
                     + " with test inputs.\nException:\n"
                     + indent(str(e))
-                )
+                ) from e
```

Comment:
Since original exception is chained now, please remove `"Expection:\n"+indent(str(e))` from the error message

Reply:
Sure.